lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -271,12 +271,14 @@ <h3 class="hd hd-3">${_("Coupon Code List")}</h3>
              url: "${section_data['sale_validation_url'] | n, js_escaped_string}",
              success: function (data) {
                  $('#error-msg').attr('class','success-msgs')
```

Comment:
I agree, but is this pre-existing code adjustment something that should be included in this PR? Happy to knock it out if so...